Makefile

```diff
@@ -62,9 +62,12 @@ json:
 	cd omas/utilities && python3 build_json_structures.py
 	make cocos
 
-cocos:
+cocos: symbols
```

Comment:
@orso82 Is this right? Seems like the next entry might've bled through here by accident, but I'm not sure.